alpine/elasticsearch/2.x/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM nowait/java:openjre8-alpine
+MAINTAINER Nowait <devops@nowait.com>
+
+ENV ELASTICSEARCH_VERSION 2.3.2
+ENV CONFD_VERSION 0.12.0-alpha3
+ENV CONFD_BACKEND env
+
+RUN \
+  apk add --no-cache ca-certificates curl su-exec bash && \
+  mkdir -p /opt && \
+  cd /tmp && \
+  curl https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/$ELASTICSEARCH_VERSION/elasticsearch-$ELASTICSEARCH_VERSION.tar.gz > elasticsearch-$ELASTICSEARCH_VERSION.tar.gz && \
+  tar -xzf elasticsearch-$ELASTICSEARCH_VERSION.tar.gz && \
+  rm -rf elasticsearch-$ELASTICSEARCH_VERSION.tar.gz && \
+  mv elasticsearch-$ELASTICSEARCH_VERSION /opt/elasticsearch && \
+  addgroup elasticsearch && \
+  adduser -SDH elasticsearch -G elasticsearch
+
+RUN \
+  cd /tmp && \
```

Comment:
Rather than download and move this file, I would just do this:
mkdir /opt && \
curl -L https://github.com/kelseyhightower/confd/releases/download/v$CONFD_VERSION/confd-$CONFD_VERSION-linux-amd64 > /opt/confd && \
chmod +x /opt/confd